app/controllers/places_controller.rb

```diff
@@ -11,6 +11,10 @@ def leaderboard
     @leaderboard = LeaderboardPresenter.new
   end
 
+  def show
+    @place = PlacePresenter.new(Place.find(params[:id]))
```

Comment:
I think I'd prefer these urls to have a slug, but this can come later

Reply:
It's true that this is how they were done in the original version, will have a look.

Reply:
https://github.com/norman/friendly_id might be worth a look